api/sso/tests/features/steps/add_account.py

```diff
@@ -0,0 +1,93 @@
+# Mycroft Server - Backend
+# Copyright (C) 2019 Mycroft AI Inc
+# SPDX-License-Identifier: 	AGPL-3.0-or-later
+#
+# This file is part of the Mycroft Server.
+#
+# The Mycroft Server is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+"""Step functions for adding an account via the single sign on API."""
+from binascii import b2a_base64
+from datetime import datetime
+
+from behave import given, then, when  # pylint: disable=no-name-in-module
+from flask import json
+from hamcrest import assert_that, equal_to, is_in, not_none
+
+from selene.data.account import AccountRepository, PRIVACY_POLICY, TERMS_OF_USE
+
+
+@given("a user completes new account setup")
+def build_new_account_request(context):
+    """Build base request data for the new account tests."""
+    context.new_account_request = dict(
+        termsOfUse=True,
+        privacyPolicy=True,
+        login=dict(
+            federatedPlatform=None,
+            federatedToken=None,
+            email=b2a_base64(b"bar@mycroft.ai").decode(),
```

Comment:
This e-mail is used in several places should it be made into a global constant or something? like TEST_EMAIL or specified in a given step? `given a user with e-mail bar@mycroft.ai`